functions/toml.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/*
+TOML Parser
+Super inefficient, but gets the job done.
```

Comment:
Yeah. All that matters is that it works currently, but this should be improved as time goes on.